src/connection/connectiontypes.js

```diff
@@ -9,10 +9,10 @@ export type ConnectionCursor = string;
  * A flow type designed to be exposed as `PageInfo` over GraphQL.
  */
 export type PageInfo = {|
-  startCursor: ?ConnectionCursor,
-  endCursor: ?ConnectionCursor,
-  hasPreviousPage: ?boolean,
-  hasNextPage: ?boolean,
+  startCursor?: ConnectionCursor | null,
+  endCursor?: ConnectionCursor | null,
+  hasPreviousPage?: boolean | null,
+  hasNextPage?: boolean | null,
```

Comment:
Can you please check if we actually need `null` or not?
If it just inputs types than we are totally ok to make it just optional.
If it's also output types please check what's actually returned.